File: front_workspace/js/5월/day0514/js/상속/Bird.js
// 모든 새들의 어버이 격인 일반적인 새
class Bird{
    constructor(color, age){
        this.color = color;
        this.age = age;

        console.log("Bird 생성자 호출");
        console.log("Bird", this.color, this.age);
    }

    eat(){
        console.log("새가 먹이 먹음");
    }
    fly(){
        console.log("새가 날음");
    }
}

File: back_workspace/java_workspace/5월/day0515/src/animal/Bird.java
package animal;


public class Bird{
	String name = "부모새";
	int age = 5;
	
	/*
	public Bird(){
		super(); // 최상위 부모클래스 Object()를 초기화		
	}
	*/
	
	public void fly(){
		System.out.println("부모새가 날아요");
	}
	
	public void eat(){
		System.out.println("밥먹어요");
	}
}